core/src/main/java/com/mapzen/android/graphics/MapInitializer.java

```diff
@@ -90,15 +91,19 @@ private void loadMap(final MapView mapView, String sceneFile, final OnMapReadyCa
     final List<SceneUpdate> sceneUpdates = sceneUpdateManager.getUpdatesFor(apiKey, locale,
         mapStateManager.isTransitOverlayEnabled(), mapStateManager.isBikeOverlayEnabled(),
         mapStateManager.isPathOverlayEnabled());
```

Comment:
I'm glad you brought this up and think it would be nice to provide more global options like this. Created tickets in the iOS and Android SDKs for future work to complete this
https://github.com/mapzen/android/issues/431
https://github.com/mapzen/ios/issues/355